create-openssl-framework.sh

```diff
@@ -232,6 +250,7 @@ else
     NORMALIZE_OPENSSL_VERSION=no
 
     for SYS in ${ALL_SYSTEMS[@]}; do
+        echo bin/${SYS}*/lib/libcrypto.a bin/${SYS}*/lib/libssl.a
```

Comment:
Let's remove such prints if they are not functional. Or at least make them more understandable for the user.